backend/test/api/scores_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns api.scores-test
+  (:require [expectations.clojure.test :refer :all]
+            [api.base :as api]
+            [test-helper :as helper]
+            [configs.hands :as hands]))
+
+(defexpect rows-won
+  (expect
+    #(= [3 1] 
+        (:scores %))
+    (helper/play-as-api
+      (fn [i] (mod i 4))
+      (fn [i] 0)))
+  (expect
+    #(= [2 2]
+        (:scores %))
+    (helper/play-as-api
+      (fn [i] (mod i 2))
+      (fn [i] (+ (mod i 2) 2)))))
+
+(defexpect power-in-rows
+  ; Game begins with scores at 0
+  (expect
+    [[0 0] [0 0] [0 0] [0 0] [0 0]]
+    (let [game-state (api/create-game)
+          game-id (:game-id game-state)
+          player-id (:player-id game-state)
+          opponent-id (:player-id (api/add-player game-id))]
+      (:rows-power (api/get-game game-id player-id))))
+  (expect
+    [[(helper/default-hand-power 0) (helper/default-hand-power 1)]
+     [0 0] [0 0] [0 0] [0 0]]
+    (let [game-state (api/create-game)
+          game-id (:game-id game-state)
+          player-id (:player-id game-state)
+          opponent-id (:player-id (api/add-player game-id))]
+      (do
+        (api/play-card-as-player game-id player-id 0 0)
+        (api/play-card-as-player game-id opponent-id 1 0))
+      (:rows-power (api/get-game game-id player-id))))
+  (expect
+    [[0 (+ (helper/default-hand-power 0) (helper/default-hand-power 1))]
```

Comment:
ok, now you're adding them together, I think the name is definitely not descriptive enough

Reply:
Since you don't understood what they do it's probably not descriptive enough. Even though I can't think of a better name (that's not way longer).

`nth-default-card-power`?